roles/hydra-stack/defaults/main.yml

```diff
@@ -4,4 +4,9 @@ solr_context: hydra
 base_path: prod
 capistrano_user: deploy
 capistrano_group: deploy
+
 fedora_ver: 4.5.0
+fedora_sha: b9781a38227b71bb2f61d09938f15d5621d57edcfb94db5e9524abcac6984b14
```

Comment:
The task specifies sha256 - a modified variable name (fedora_256 or fedora_sha256) or a comment mentioning the constraint would be useful for anyone who wants to override the defaults and also a breadcrumb for Future Me if/when Fedora/Solr move to SHA-3.